ScenePlayer/res/layout/recording_player.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-	xmlns:rjdj="http://schemas.android.com/apk/res/me.rjdj.rjdjroid"
```

Comment:
Right. In this case it's not used at all in that file, that's why I removed
it.

On Wed, 28 Oct 2015 19:59 Joe Bowbeer <notifications@github.com> wrote:

> In ScenePlayer/res/layout/recording_player.xml
> <https://github.com/joebowbeer/pd-for-android/pull/2#discussion_r43302172>
> :
>
> > @@ -1,6 +1,5 @@
> >  <?xml version="1.0" encoding="utf-8"?>
> >  <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
> > -	xmlns:rjdj="http://schemas.android.com/apk/res/me.rjdj.rjdjroid"
>
> If xmlns is needed anywhere for something similar, it should point to the
> generic "res-auto" schema for gradle-build compatibility.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joebowbeer/pd-for-android/pull/2/files#r43302172>.
>